README.md

```diff
@@ -1 +1,11 @@
-# LifeGame
\ No newline at end of file
+# LifeGame
+
+[![Build Status](https://travis-ci.org/MMorihiro/LifeGame.svg?branch=20-travis.yml)](https://travis-ci.org/MMorihiro/LifeGame) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
Oh, I also want to add it if the coverage is not bad.
Since this is a game, the code which can be tested is few and the coverage is so bad that I think it is not useful.(just `2%`) Therefore, I didn't add the badge.

Using Codecov is practice for config4k.
This is why I decided to measure coverage.
I want to use Codecov badge in config4k.